iree/compiler/Dialect/HAL/Target/LLVM/LLVMTargetOptions.cpp

```diff
@@ -107,6 +107,12 @@ LLVMTargetOptions getLLVMTargetOptionsFromFlags() {
       llvm::cl::init(llvmTargetOptions.debugSymbols));
   llvmTargetOptions.debugSymbols = clDebugSymbols;
 
+  static llvm::cl::opt<bool> clAddressSanitizer(
+      "iree-llvm-address-sanitizer",
```

Comment:
If you could change this to `iree-llvm-sanitize` as an enum value then we can better match the compiler ASAN flags people are used to and support other sanitizers in the future (`-fsanitize=address` -> `-iree-llvm-sanitize=address`, =memory, etc etc)

Reply:
Thanks Ben, I updated this part in my new commit. Now address sanitizer is turned on by -iree-llvm-sanitizer=address.
But, for that, I used the member variable, sanitizerKind, representing sanitizer modes in LLVMTargetOption as int type, and used a flag in flatBuffer as ubyte. Is it okay? or can I use other data type for those?